tools/run_tests/dockerize/build_docker_and_run_tests.sh

```diff
@@ -82,6 +82,7 @@ docker run \
   -e "BUILD_URL=$BUILD_URL" \
   -e "JOB_BASE_NAME=$JOB_BASE_NAME" \
   -i $TTY_FLAG \
+  --sysctl net.ipv6.conf.all.disable_ipv6=0 \
```

Comment:
do we still need this after setting docker options? I'd rather not add this here unless absolutely necessary.